libsolidity/ast/AST.h

```diff
@@ -1019,6 +1024,7 @@ class FunctionDefinition: public CallableDeclaration, public StructurallyDocumen
 	StateMutability m_stateMutability;
 	bool m_free;
 	Token const m_kind;
+	bool m_usableAsSuffix;
```

Comment:
I know that this is fine and that there's also ``m_free`` above and everything... still I'm also getting itchy when I see POD-members that aren't directly member-initialized :-).
I'd *always* do a
```suggestion
	bool m_usableAsSuffix = false;
```
for *all* cases like this - if this is actually assigned to in the constructor this has no effect whatsoever (i.e. also zero penalty) - but in case this actually *isn't* initialized in a (potentially future) constructor, this will change undefined behaviour into deterministic behaviour...

But yeah - just wanted to remark this generally, both changing it here and keeping it is ultimately fine with me.

Reply:
Added default initialization for all members.